docs/internal/release-procedure.md

```diff
@@ -42,7 +46,13 @@ git push origin <YOUR_VERSION>
 
 Pushing a tag will trigger a **GitHub Actions** workflow on the `main` branch. Check the "**Arduino IDE**" workflow and see that everything goes right. If the workflow succeeds, a new release will be created automatically and you should see it on the ["**Releases**"](https://github.com/arduino/arduino-ide/releases) page.
 
-## 📄 Create the changelog
+### 4. ⬆️ Bump version metadata of packages
+
+In order for the version number of the tester and nightly builds to have correct precedence compared to the release version, the `version` field of the project's `package.json` files must be given a patch version bump (e.g., `2.0.1` -> `2.0.2`) **after** the creation of the release tag.
```

Comment:
In the long run, we should use `yarn` to generate these pre-release version numbers instead of manually bumping them. Once this is merged, I will create a follow-up with the request.

Ref: https://classic.yarnpkg.com/en/docs/cli/version/#toc-yarn-version-prerelease